packages/types/src/Signers.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+export type SignerInterface = {
+  algorithm: string
+  id: string
```

Comment:
I think it could potentially be confusing for a developer to determine what string they should enter here as the ID, especially when signers can be both accounts and DIDs, depending on the context . But I don't really have a suggestion for improvement, other than making sure we have good documentation.

Reply:
Of course if they always use the `getSigner` methods then they don't even have to care what the ID is.

Reply:
I think that's the intended way for the signer to be created, indeed.

Reply:
yeah this would mostly get unnoticed; there may be a `getAccountSigners` function or method that creates all signers for an account (using Addresses as ids) and a `getSigners` method on a DID class or something that returns all signers for its verificationMethods (where all ids are DID URLs). This higher level api is TBD in a follow-up PR, but I see a lot of potential